actionlist/app/containers/PreferencesContainer.js

```diff
@@ -0,0 +1,271 @@
+              <View style={{ backgroundColor: rowData }}><Text>{rowData}</Text></View>}
+            onSelect={(idx, value) => onSelectOption(idx, value, 'approved')}
+          />
+        </View>
+      </View>
+      <View style={styles.preferenceSelect}>
+        <View>
+          <Text style={styles.text}>Final</Text>
+        </View>
+        {/* Disabled Following ES-Lint rule
+          It is corrected in latest version which is still in beta */}
+        <View>
+          <ModalDropdown
+            style={[styles.dropdown,{backgroundColor: dropdownColors.final}]} options={OPTIONS} renderRow={(rowData) => // eslint-disable-line
+              <View style={{ backgroundColor: rowData }}><Text>{rowData}</Text></View>}
+            onSelect={(idx, value) => onSelectOption(idx, value, 'final')}
+          />
+        </View>
+      </View>
+      <View style={styles.preferenceSelect}>
+        <View>
+          <Text style={styles.text}>Processed</Text>
+        </View>
+        {/* Disabled Following ES-Lint rule
+          It is corrected in latest version which is still in beta */}
+        <View>
+          <ModalDropdown
+            style={[styles.dropdown,{backgroundColor: dropdownColors.processed}]} options={OPTIONS} renderRow={(rowData) => // eslint-disable-line
+              <View style={{ backgroundColor: rowData }}><Text>{rowData}</Text></View>}
+            onSelect={(idx, value) => onSelectOption(idx, value, 'processed')}
+          />
+        </View>
+      </View>
+      <View style={styles.preferenceSelect}>
+        <View>
+          <Text style={styles.text}>Exception</Text>
+        </View>
+        {/* Disabled Following ES-Lint rule
+          It is corrected in latest version which is still in beta */}
+        <View>
+          <ModalDropdown
+            style={[styles.dropdown,{backgroundColor: dropdownColors.exception}]} options={OPTIONS} renderRow={(rowData) => // eslint-disable-line
+              <View style={{ backgroundColor: rowData }}><Text>{rowData}</Text></View>}
+            onSelect={(idx, value) => onSelectOption(idx, value, 'exception')}
+          />
+        </View>
+      </View>
+      <View style={styles.preferenceSelect}>
+        <View>
+          <Text style={styles.text}>Cancel</Text>
+        </View>
+        {/* Disabled Following ES-Lint rule
+          It is corrected in latest version which is still in beta */}
+        <View>
+          <ModalDropdown
+            style={[styles.dropdown,{backgroundColor: dropdownColors.cancel}]} options={OPTIONS} renderRow={(rowData) => // eslint-disable-line
+              <View style={{ backgroundColor: rowData }}><Text>{rowData}</Text></View>}
+            onSelect={(idx, value) => onSelectOption(idx, value, 'cancel')}
+          />
+        </View>
+      </View>
+    </View>
+    <View style={styles.preferences_buttons}>
+      <View style={styles.buttonContainer}>
```

Comment:
These `Button`s can be extracted to a utility component so you're not repeating so much code.